interfaces/PA-App/src/app/personal-components/preprinted-qrcode/preprinted-qrcode.component.ts

```diff
@@ -76,7 +76,9 @@ export class PreprintedQrcodeComponent extends PersonalComponent {
   async checkValidation() {
     if (!this.program.validation) {
       this.cancel();
+      return;
     }
+    this.isCanceled = false;
```

Comment:
@elwinschmitz there are 3 more components (besides these 2) where "isCanceled === false" is used. I haven't  touched those, because nothing is going wrong apparently. Feel free to touch those as well if you think it's needed though.

Reply:
@jannisvisser Some of those components have a line of `this.isCanceled = false` already somewhere else in the code. I think that was where this component was 'out of line'..